Changes

```diff
@@ -160,6 +160,9 @@ Working version
 
 ### Standard library:
 
+- #12131: Simplify implementation of weak hash sets. (Nick Barnes,
+  review by ??).
```

Comment:
You should add the reviewers too and move it to the 5.1 section. Moreover, I would propose to explain in more detail that is an important performance regression fix (or alternatively to move the change entry to the bug fix section).

Reply:
There isn't a 5.1 section in this `Changes`; I guess I could do a catch-up merge?

Reply:
For the record, I did this catch-up merge and sorted the Changes entry.